system/include/webgpu/webgpu.h

```diff
@@ -826,6 +868,7 @@ typedef struct WGPUShaderModuleSPIRVDescriptor {
 typedef struct WGPUShaderModuleWGSLDescriptor {
     WGPUChainedStruct chain;
     char const * source;
+    char const * code;
```

Comment:
Hmm... this ended up with both `source` and `code`. This is because the Dawn generator includes things tagged as both "emscripten" and "upstream" in this output which I'm certain something else relied on, but probably we need to not include "upstream" things in the emscripten generation, here:
https://source.chromium.org/chromium/chromium/src/+/main:third_party/dawn/generator/dawn_json_generator.py;l=850;drc=ec9cf2a85c2a8266478182bae48f490d4e94c5fb
And update a few items in dawn.json to explicitly include the "emscripten" tag.